packages/visualizations-react/stories/Chart/StudioLayouts/PieChart.stories.tsx

```diff
@@ -140,3 +140,48 @@ const PieTitleLegendValuesArgs: Props<DataFrame, ChartOptions> = {
     },
 };
 PieTitleLegendValues.args = PieTitleLegendValuesArgs;
+
+export const PieDataLabelCustomLegendValues = ChartTemplate.bind({});
+const PieDataLabelCustomLegendValuesArgs: Props<DataFrame, ChartOptions> = {
+    data: {
+        loading: false,
+        value: [
+            { x: 'Alpha', y: 100 },
+            { x: 'Beta', y: 50 },
+            { x: 'Gamma', y: 20 },
+            { x: 'Delta', y: 30 },
+        ],
+    },
+    options: {
+        labelColumn: 'x',
+        source: defaultSource,
+        series: [
+            {
+                type: ChartSeriesType.Pie,
+                valueColumn: 'y',
+                backgroundColor: ['#CB4335', '#1F618D', '#F1C40F', '#27AE60'],
+                dataLabels: {
+                    display: true,
+                },
+            },
+        ],
+        padding: 32,
+        legend: {
+            display: false,
+            custom: true,
+            position: 'right',
+            align: 'center',
```

Comment:
If I'm not mistaken, I haven't seen the option align used in the custom legend ?